Makefile

```diff
@@ -6,14 +6,25 @@ GOCLEAN=$(GOCMD) clean
 GOTEST=$(GOCMD) test
 GOGET=$(GOCMD) get
 GOMOD=$(GOCMD) mod
+GOFMT=$(GOCMD) fmt
 
 .PHONY: all test coverage
 all: test coverage
 
+checkfmt:
```

Comment:
@filipecosta90 While this will work, an easier way of linting in go would be using https://github.com/golangci/golangci-lint

Reply:
@danni-m I've used this `go fmt` approach as a quick fix/quick enforcing of fmt. Will revise the required changes to use golangci-lint and move towards that afterwards. wdyt of adding this quick check to be safeguarded and with more time moving to something more correct/sophisticated?

Reply:
@filipecosta90 sounds good to me, just want to make sure you are aware that this tool exists.